### DISCUSSION TITLE
Need translation feature

### DISCUSSION THREAD

**Discussion by axsweet** [community/community#16873] [Mobile]
Add method to translate issues and replies. Even read mes. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.65.0 (67507275)
Device: iPhone 13 Pro (iOS 15.5)
</details>

**Comment by brianlovin** [comment]
Thanks for flagging! Will share this with our team now to see what's possible 😊

**Reply by TPS** [reply]
Integrating Bing's autotranslate would fix this
